usr.sbin/devinfo/devinfo.c

```diff
@@ -213,34 +309,53 @@ print_rman_resource(struct devinfo_res *res, void *arg __unused)
 	struct devinfo_dev	*dev;
 	struct devinfo_rman	*rman;
 	rman_res_t		end;
+	char			*res_str, *entry;
 	bool			hexmode;
 
 	dev = devinfo_handle_to_device(res->dr_device);
 	rman = devinfo_handle_to_rman(res->dr_rman);
 	hexmode =  (rman->dm_size > 1000) || (rman->dm_size == 0);
 	end = res->dr_start + res->dr_size - 1;
 
-	printf("    ");
-
 	if (hexmode) {
 		if (res->dr_size > 1)
-			printf("0x%jx-0x%jx", res->dr_start, end);
+			asprintf(&res_str, "0x%lx-0x%lx", res->dr_start, end);
```

Comment:
btw res->dr_start is a `*max_t` (uintmax_t) and %j seems to be standard prefix to format a uintmax_t
but i'm not too sure about it

Reply:
Fixed. Not shure how I messed that up. It was right to begin with. Good catch!